pkg/destroy/aws/shared.go

```diff
@@ -242,6 +242,10 @@ func (o *ClusterUninstaller) cleanSharedHostedZone(ctx context.Context, client *
 					}
 					lastError = errors.Wrapf(err, "deleting record set %#v from zone %s", recordSet, id)
 				}
+				recordSetLogger := logger.WithField("id", id).WithField(
+					"recordset",
+					fmt.Sprintf("%s (%s)", aws.StringValue(recordSet.Name), aws.StringValue(recordSet.Type)),
+				)
 				recordSetLogger.Debug("Deleted")
```

Comment:
Assuming that we've already set the "id" logger field (which we want to do in `cleanSharedRoute53`,
```suggestion
				logger.WithField(
					"recordset",
					fmt.Sprintf("%s (%s)", aws.StringValue(recordSet.Name), aws.StringValue(recordSet.Type)),
				).Debug("Deleted")
```